Source/JavaScriptCore/wasm/js/JSWebAssemblyException.cpp

```diff
@@ -74,8 +78,18 @@ void JSWebAssemblyException::destroy(JSCell* cell)
 
 JSValue JSWebAssemblyException::getArg(JSGlobalObject* globalObject, unsigned i) const
 {
-    ASSERT(i < tag().type().argumentCount());
-    return toJSValue(globalObject, tag().type().argumentType(i), payload()[i]);
+    const auto& tagType = tag().type();
+    ASSERT(i < tagType.argumentCount());
+
+    // It feels like maybe we should throw an exception here, but as far as I can tell,
+    // the current draft spec just asserts that we can't getArg a v128. Maybe we can
+    // revisit this later.
+    RELEASE_ASSERT(tagType.argumentType(i).kind != Wasm::TypeKind::V128);
```

Comment:
I saw https://github.com/WebAssembly/exception-handling/issues/308, nice.

I wonder if we can send this over to the formal verification folks, this seems like the kind of thing that should have been able to be caught.

Either way, we shouldn't crash IMO. This might just block the fuzzer